packages/carbon-graphs/src/js/controls/PairedResult/helpers/helpers.js

```diff
@@ -683,7 +683,7 @@ const clickHandler = (graphContext, control, config, canvasSVG) => (
   const pairedBoxGroup = d3.selectAll(`.${styles.pairedBoxGroup}`);
   pairedBoxGroup.each(function () {
     const clipPath = d3.select(this).attr('clip-path');
-    if (clipPath === pairedBoxGroupClipPath) {
```

Comment:
No unit tests?

Reply:
No veera. There were already some unit tests to check those lines during the toggle.

Reply:
Even though there are lines to verify this logic, it likely does not test it with the presence of the quotes.  Can we add a test that mocks the clip-path returning a value with the quotes as seen in IE so that we have a test which covers both values which we know might be returned by browsers?

Reply:
Added [here](https://github.com/cerner/terra-graphs/pull/30/commits/1ed69c43bc33aa00c4da32091e67acc55bfa3e66#diff-a1e1e5d218ab7da13636fbc6cce4a15cee9ab2948153df45ae61604bbeafd4afR2313).